src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -9219,6 +9219,10 @@ static void Main()
                 // CS0151ERR_IntegralTypeValueExpected}
                 Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "}").WithArguments("}").WithLocation(8, 36)
                 );
+
+            Assert.Equal("7", comp.GetRequiredLanguageVersion(comp.GetDiagnostics()[0]));
+            Assert.Null(comp.GetRequiredLanguageVersion(comp.GetDiagnostics()[2]));
+            Assert.Throws<ArgumentNullException>(() => comp.GetRequiredLanguageVersion(null));
```

Comment:
I'll probably leave this one as-is. I don't want to set up a new compilation just for this one line.